js/src/components/tree-select-control/options.js

```diff
@@ -86,7 +91,14 @@ const Options = ( {
 				</Flex>
 
 				{ hasChildren && isExpanded && (
-					<div className="woocommerce-tree-select-control__children">
+					<div
+						className={ classnames(
+							'woocommerce-tree-select-control__children',
+							isRoot( option )
+								? 'woocommerce-tree-select-control__main'
```

Comment:
❔ 💅 `"'woocommerce-tree-select-control__main'"` CSS class is used when `isRoot` is true, so maybe consider renaming the CSS class to `"woocommerce-tree-select-control__root"`? 🤔

Reply:
I had the same thought about that on development. 

I went to __main because root well evolves as well the "Select all" option. Hence, I considered __main as the main group of options a part of the root.